README.md

```diff
@@ -37,6 +37,12 @@ Become a sponsor and get your logo on our README on Github with a link to your s
 
 ## Packages
 
+***Note:** The links below will take you to the master branch version of the docs.
```

Comment:
I'm up for this change, but it's easier to browse via `tags`.

![image](https://user-images.githubusercontent.com/28659384/46622174-ddcb8380-cb29-11e8-8e2c-0ce4e70db1d7.png)

Reply:
@timdeschryver right you are! Didn't know about that option. I updated the suggestion text.